examples/formbot/myactions.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from rasa_core_sdk import Action
+from rasa_core_sdk.events import SlotSet, StartForm, EndForm
+from rasa_core_sdk.forms import SimpleForm
+
+
+class StartFormAction(Action):
+    def name(self):
+        return "start_restaurant"
+
+    def run(self, dispatcher, tracker, domain, executor):
+        return [StartForm("restaurant_form")]
+
+
+class EndFormAction(Action):
+    def name(self):
+        return "end_form"
+
+    def run(self, dispatcher, tracker, domain, executor):
+        form = executor.forms[tracker.active_form]
+        still_to_go = form.check_unfilled_slots(tracker)
+        complete = len(still_to_go) == 0
+        return [SlotSet('form_complete', complete), EndForm()]
+
+
+class RestaurantForm(SimpleForm):
+    def __init__(self):
+        name = 'restaurant_form'
```

Comment:
all of these are static values, why not make them class attributes?

Reply:
they are validated and set as attribjutes in the __init__ for SimpleForm